### PR TITLE
README.md: Update documentation links

### PR DESCRIPTION
**Pull request by alexbrand**
README.md: Update documentation links

### CODE REVIEW

**Comment by wwitzel3** [comment]
Thanks @alexbrand I like this change, would you mind doing it for the rest of the doc links in the readme?

**Comment by alexbrand** [comment]
Will do!

File: README.md [hunk 1]
```diff
@@ -124,15 +124,15 @@ Octant should immediately launch your default web browser on `127.0.0.1:7777`.
 Or to run it on a specific host and fixed port:
 `$ OCTANT_LISTENER_ADDR=0.0.0.0:8900 octant`
 
-For configuring Octant, setting up a development environment, or running tests, refer to the documentation [here](site/docs/master/getting-started.md).
+For configuring Octant, setting up a development environment, or running tests, refer to the documentation [here](https://octant.dev/docs/master).
```
Comment (wwitzel3) [inline README.md]: can we just make documentation the link?

Comment (alexbrand) [reply README.md]: Good call! Fixed.

**Pull request merged**